pandas/core/series.py

```diff
@@ -347,7 +360,11 @@ def _init_dict(self, data, index=None, dtype=None):
             keys, values = [], []
 
         # Input is now list-like, so rely on "standard" construction:
-        s = Series(values, index=keys, dtype=dtype)
+        # TODO: warning filter can be removed when default dtype for Series
```

Comment:
Is this filter to avoid a double warning?

Reply:
Yes indeed, otherwise `Series({})` is raising the new FutureWarning twice